src/stories/DataDisplay/DateRangePicker.stories.js

```diff
@@ -38,16 +38,11 @@ const DateRangePickerStories = {
 const today = dayjs("2022-10-10").toDate();
 
 export const Default = () => {
-    const [value, setValue] = useState({ from: new Date("2022-02-03"), to: new Date("2022-03-08") });
```

Comment:
This was a standard date picker without range - copy/paste from the main story, so removed it.